website/docs/users-sources/sources/social-logins/google/workspace/index.md

```diff
@@ -0,0 +1,197 @@
+
+---
+
+## Integration
+
+By the end of this integration, your authentik instance will allow users to authenticate using their Google Workspace credentials.
+
+You'll need to have authentik instance running and accessible on an HTTPS domain, and a Google Workspace domain with super-administrator access.
+
+Keep a text-editor handy because we'll be copying and pasting values between the two services.
+
+:::info
+
+The following placeholders are used in this guide:
+
+|                     |                                                           |
+| ------------------- | --------------------------------------------------------- |
+| `authentik.company` | The Fully Qualified Domain Name of the authentik install. |
+
+:::
+
+## Google Workspace Configuration
+
+We'll need some information from Google to complete the integration, so we'll
+start by logging into [Workspace Admin Console](https://admin.google.com/) as a super-admin.
+
+### Creating a new app
+
+From the Workspace Admin Console, navigate to the _Apps_ section, and then to _Web and mobile apps_.
+Continue by expanding the **Add app** dropdown and selecting **Add custom SAML app.**
+
+Within the app creation page, fill in the following fields with your desired values:
+
+| Field       | Value                        |
+| ----------- | ---------------------------- |
+| Name        | authentik                    |
+| Description | Single Sign-On for authentik |
+
+### Google Identity Provider details
+
+Next, you'll be presented with information we'll need to configure authentik. Under _Option 2_, copy the SSO URL and download the certificate.
+
+:::warning
+authentik is acting as both a Service Provider (SP) to Google and an Identity Provider (idP) to your applications. Since we only need the SP configuration, you can ignore the Entity ID provided by Google.
+
+|           |                                                           |
+| --------- | --------------------------------------------------------- |
+| SSO URL   | `https://accounts.google.com/o/saml2/idp?idpid=#########` |
+| Entity ID | ~~`https://accounts.google.com/o/saml2?idpid=#########`~~ |
+
+:::
+
+### Service Provider details
+
+We'll need to provide Google with some information about our authentik instance, specifically the Assertion Consumer Service (ACS) URL. This URL is where Google will send the SAML response after a user has authenticated. We'll also need to provide the Entity ID, which can be any unique identifier, but we recommend using the URL of your authentik instance.
+
+| Field           | Value                                               |
+| --------------- | --------------------------------------------------- |
+| ACS URL         | `https://authentik.company/source/saml/google/acs/` |
+| Entity ID       | `https://authentik.company`                         |
+| Name ID format  | EMAIL                                               |
```

Comment:
IS EMAIL the literal value that they enter? For Name ID, what do they enter? (Also we need to say "email address" not just "email".)